src/apps/lwaftr/lwaftr.lua

```diff
@@ -428,6 +447,9 @@ function LwAftr:new(conf)
    o.icmpv6_error_count = 0
    o.icmpv6_error_rate_limit_start = 0
 
+   o.next_alarm_request_time = engine.now()
```

Comment:
Can we package this into an object?  Most of these alarms are going to have the same structure, so it makes sense to re-use code.  i.e.

```lua
local CounterAlarm = {}
function CounterAlarm.new(alarm, period, limit, object, counter)
   return setmetatable({alarm=alarm, period=period, limit=limit, object=object, counter=counter},
                                   {__index = CounterAlarm})
end
function CounterAlarm:check()
   if self.next_check == nil then
      self.next_check = engine.now() + self.period
      self.last_value = counter.read(self.object.shm[name])
   elseif engine.now() < self.next_check then
      local value = counter.read(self.object.shm[name])
      if (value - last_value > self.limit) then
         self.alarm:raise()
      else
         self.alarm:clear()
      end
      self.next_check = engine.now() + self.period
      self.last_value = value
   end
end

...

o.bad_softwire_matches_alarm = CounterAlarm.new(
   bad_softwire_matches, 5, 1e5, o, 'drop-no-dest-softwire-ipv4-packets')

...

-- In the pull() function
o.bad_softwire_matches_alarm:check()